### ISSUE TITLE
cannot use in zotero 7

### ISSUE THREAD

**Issue by a1576799** [Zar-rok/Zotero-Add-Collection-Tag#3]
Is there any chance you could revise the add-on to make it work in zotero 7? Thanks a lot!

**Comment by Zar-rok** [comment]
I'll check how to do that!

**Comment by a1576799** [comment]
Yeah, please do. It is a quite useful add-on, even better than other similar add-on like action and tag. I think your add-on is neat and essential.  Please Let me know after you update it.  Thank you so much.

**Comment by Zar-rok** [comment]
I managed to adapt the plugin for both Zotero 6 and 7.

Can you test the extension to check if it works for you too?

It's on this branch: https://github.com/Zar-rok/Zotero-Add-Collection-Tag/tree/update_to_7.0.3

Then:
- `7z a zotero_add_collection_tag.xpi bootstrap.js manifest.json install.rdf`
- Or, `zip zotero_add_collection_tag.xpi bootstrap.js manifest.json install.rdf`

**Comment by a1576799** [comment]
It works. Thank you very much!

**Comment by a1576799** [comment]
Hi, there, I just noticed that it doesn't work anymore. I added a file and it generated a item, but there was no collection tag added. Why this issue happened? Could you please help me? Thanks.

**Comment by Zar-rok** [comment]
Mhm, it's still working fine for me with Zotero `7.0.3`.

- Do you have a newer version of Zotero?
- Do you use other plugins? (Some may conflict with this repository's plugin, e.g., #2 )

**Comment by a1576799** [comment]
![image](https://github.com/user-attachments/assets/d9282927-ae30-4edc-bb2d-c50c61b355bb)
I dont use the other tag plugin, either.
Not sure about what made this happen...

**Comment by Zar-rok** [comment]
So, I've updated Zotero from version `7.0.3` to `7.0.5`.
Then, the plugin is not working (without any error messages).

I went to `Tools → Plugins → Gear icon (top right) → Install Plugin From File` and reinstalled the plugin. It has fixed the problem for me.

Can you replicate this to see if it fixes your problem?
